.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Release
+about: Announce and track an upcoming release
+labels:
+  - release
+---
+
+|  |  |
+| --- | --- |
+| Release version | YYYY.MM.X |
+| Planned release date | YYYY-MM-DD |
```

Comment:
Do we need to worry about a timezone here (given it could be off +/-1 day)?

Reply:
I don't think timezone considerations will be will a big deal for this template (others should feel free to chime in if I'm missing something)

Reply:
Probably not enough to include in the date itself. Though maybe we could say something like this?

```suggestion
| Planned release date (US Central) | YYYY-MM-DD |
```

No strong feelings about what timezone goes there. Could see a case for US Eastern or similar.

Reply:
Releases often happen after the end of my work day. But I generally expect that so not sure if there is any real value in being explicit here.